src/themes/elife/custom-properties.css

```diff
@@ -0,0 +1,5 @@
+:--root {
+  --text-size-rem-small: 0.6875;
+  --letter-spacing: 0.5px;
+  --line-height-calc: 1.5rem;
```

Comment:
This represents the standard baseline grid measure on eLife (24px), expressed in rem, so would name it something like `--baseline-measure-rem`